docs-src/real_time_messaging.rst

```diff
@@ -11,6 +11,10 @@ and it supports most of the event types in the RTM API. If you'd like to use the
 
 The RTMClient allows apps to communicate with the Slack Platform's RTM API.
 
+Events using the RTM API **must** use a classic Slack app (with a plain `bot` scope). If you already have a classic Slack app, you can use those credentials. 
+If you don't and need to use the RTM API, you can [**Create a Classic Slack App**](https://api.slack.com/apps?new_classic_app=1). You can learn more in the 
```

Comment:
unfortunately, this is not a markdown doc...

Reply:
This is for the real documentation, not the getting started tutorial. We generate `.rst` from `docs-src` for https://slack.dev/python-slackclient/

Reply:
Ah, I should have mentioned this clearly but there is an issue with the link.
<img width="686" alt="Screen Shot 2020-01-21 at 16 41 20" src="https://user-images.githubusercontent.com/19658/72855626-f0a98f00-3c6c-11ea-83d9-4c08b0c94147.png">

Reply:
Ah yeah you're right 😓- just fixed this